### ISSUE TITLE
Move tool is unusable after pressing arrow

### ISSUE THREAD

**Issue by KoBeWi** [godotengine/godot#38894]
**Godot version:**
3.2.1

**Steps to reproduce:**
1. Go to 2D editor
2. Select some 2D node
3. Select move tool (W)
4. Move the node
5. Press arrow
6. Try to move the node again
7. You can't
8. You need to unselect the node
9. And select it again
10. Ugh

**Comment by Anutrix** [comment]
The node is probably losing focus when arrow key is pressed.

**Comment by KoBeWi** [comment]
Seems like it doesn't happen with select tool.

**Comment by codevenkat** [comment]
Hello, I have just begun with godot development. Could I attempt a fix for this bug?

**Comment by aaronfranke** [comment]
@codevenkat Yes, and you don't need to ask.

**Comment by groud** [comment]
> @codevenkat Yes, and you don't need to ask.

It's better if she/he does say she/he is working on it though, to be sure no one will start working on it at the same time. :)

**Comment by KoBeWi** [comment]
Nah. Some people ask, get "allowed" to do PR and then never do it and never reply again. Same with people who said "I'm working on this"... few weeks ago. Just ignore them and if you want to fix the issue, fix it xd

If issue is older than few days, there's a very very low probability that you'd create a PR at the same time as another person (and if someone was first, you will see it in the issue). And even if this happens, it's possible that two PRs take the issue in different way and yours might be better. In worst case, you will at least gain some coding/engine experience.

tl;dr asking is useless

**Comment by akien-mga** [comment]
Well to clarify: asking if one can fix a bug is not necessary. Mentioning that one is working on a fix is however a good idea, to avoid having others work on it in parallel. But indeed it's common to see people comment that they're working on it, then they give up and never update their comment, so once should not be blocked by such statement if it seems old and not PR was made. Yet it's always possible to ask said commenter if they're still working on it/need help.

TL;DR: Communication is always good.

**Comment by dreamsComeTrue** [comment]
OK, I have decided to resolve your discussion :)
